inductiva/resources/machines.py

```diff
@@ -0,0 +1,112 @@
+"""Containes MachineGroup class to manage GC resources."""
+from absl import logging
+from typing import Optional
+import uuid
+import inductiva
+from inductiva import api
+from inductiva import resources
+
+from inductiva.client import ApiClient, ApiException
+from inductiva.client.apis.tags.instance_api import InstanceApi
+from inductiva.client.model.instance import Instance
+from inductiva.client.model.instance_group import InstanceGroup
+
+
+class MachineGroup():
+    """Class to manage Google Cloud resources."""
+
+    def __init__(
+        self,
+        machine_type: str,
+        num_machines: int = 1,
+        spot: bool = False,
+        disk_size_gb: int = 30,
+        label: str = None,
```

Comment:
I think you mean `label: Optional[str] = None` here.

Reply:
Yes!